### DISCUSSION TITLE
tilde (~) doesn't work with cd

### DISCUSSION THREAD

**Discussion by Mr-Ingo** [cmderdev/cmder#2698] [Q&A] [answered]
tilde (~) doesn't work with cd
![Image 017](https://user-images.githubusercontent.com/40074453/168140925-e7d10494-688b-48eb-97b3-ba89070b6383.jpg)
How to fix it?

**Comment by DRSDavidSoft** [comment, marked as answer]
In recent versions of Cmder, you can type `cd ~/.ssh` for example, then press <kbd>Tab ↹</kbd> to autocomplete and/or expand the path.

Alternatively, you can use a small utility I made called `excd` and alias it to `cd`, so the command `cd ~/.ssh` would work as expected.

See here: https://github.com/cmderdev/cmder/issues/1051#issuecomment-702983948
